app/models/email_log.rb

```diff
@@ -22,6 +22,17 @@ class EmailLog < ActiveRecord::Base
 
   scope :bounced, -> { where(bounced: true) }
 
+  scope :addressed_to_user, ->(user) do
+    where(<<~SQL, user_id: user.id)
+      EXISTS(
+        SELECT 1
+        FROM user_emails
+        WHERE user_emails.user_id = :user_id AND
+         email_logs.to_address ILIKE '%' || user_emails.email || '%'
```

Comment:
Isn't `email_logs.to_address` an email here? Why do we need to use an `ILIKE` matcher here?

Reply:
You're right, I forgot that EmailLog just has an individual address instead of multiple like IncomingEmail